### PR TITLE
chore: update sqlx to 0.7.1

### PR DESCRIPTION
**Pull request by sebastienverreault**

### CODE REVIEW

File: ledger/Cargo.toml [hunk 1]
```diff
@@ -11,14 +11,20 @@ fail-on-warnings = []
 [dependencies]
 shared = { path = "../shared", package = "stablesats-shared" }
 
-sqlx-ledger = "0.5.12"
+sqlx-ledger = { git = "https://github.com/GaloyMoney/sqlx-ledger", branch = "main", features = ["otel"] }
```
Comment (thevaibhav-dixit) [inline ledger/Cargo.toml:14]: Why are we referencing the git branch? The version on crates.io should work ?

File: shared/Cargo.toml [hunk 2]
```diff
@@ -23,8 +26,17 @@ rust_decimal_macros = "1.29.0"
...
-sqlxmq = { version = "0.4.1", default-features = false, features = [ "runtime-tokio-rustls" ] }
+sqlx = { version = "0.7.1", features = [
+    "runtime-tokio-rustls",
+    "postgres",
+    "rust_decimal",
+    "uuid",
+    "chrono",
+    "json",
+] }
+sqlxmq = { git = "https://github.com/Diggsey/sqlxmq", rev = "658d584", default-features = false, features = [
```
Comment (thevaibhav-dixit) [inline shared/Cargo.toml:37]: we forked this for bria. I think we should use it here. 
`sqlxmq = { git = "https://github.com/GaloyMoney/sqlxmq", rev = "f03f73c", default-features = false, features = ["runtime-tokio-rustls"] }`

File: user-trades/Cargo.toml [hunk 3]
```diff
@@ -15,8 +15,8 @@ galoy_client = { path = "../galoy-client", package = "galoy-client" }
 thiserror = "1.0.40"
 chrono = "0.4"
 tokio = "1.29.1"
-sqlx = { version = "0.6", features = [ "offline", "runtime-tokio-rustls", "postgres", "decimal", "uuid", "chrono", "json" ] }
-sqlxmq = { version = "0.4.1", default-features = false, features = [ "runtime-tokio-rustls" ] }
+sqlx = { version = "0.7.1", features = [ "runtime-tokio-rustls", "postgres", "rust_decimal", "uuid", "chrono", "json" ] }
+sqlxmq = { git = "https://github.com/Diggsey/sqlxmq", rev = "658d584", default-features = false, features = [ "runtime-tokio-rustls" ] }
```
Comment (thevaibhav-dixit) [inline user-trades/Cargo.toml:19]: We can use the fork here as well

**Review by thevaibhav-dixit — COMMENTED** [review]
Few suggested changes otherwise looks good.

File: shared/Cargo.toml [hunk 4]
```diff
@@ -23,8 +26,17 @@ rust_decimal_macros = "1.29.0"
 serde = { version = "1.0.158", features = ["derive"] }
 serde_json = "1.0.93"
 serde_with = "2.3.1"
-sqlx = { version = "0.6", features = [ "offline", "runtime-tokio-rustls", "postgres", "decimal", "uuid", "chrono", "json" ] }
-sqlxmq = { version = "0.4.1", default-features = false, features = [ "runtime-tokio-rustls" ] }
+sqlx = { version = "0.7.1", features = [
```
Comment (thevaibhav-dixit) [inline shared/Cargo.toml:29]: can you format this so that everything is in a single line.

File: shared/Cargo.toml [hunk 5]
```diff
@@ -23,8 +26,17 @@ rust_decimal_macros = "1.29.0"
...
+    "runtime-tokio-rustls",
+    "postgres",
+    "rust_decimal",
+    "uuid",
+    "chrono",
+    "json",
+] }
+sqlxmq = { git = "https://github.com/GaloyMoney/sqlxmq", rev = "f03f73c", default-features = false, features = [
```
Comment (thevaibhav-dixit) [inline shared/Cargo.toml:37]: same for this

File: shared/Cargo.toml (hunk 4 above)
Comment (sebastienverreault) [reply shared/Cargo.toml]: done

File: shared/Cargo.toml (hunk 5 above)
Comment (sebastienverreault) [reply shared/Cargo.toml]: and done

**Review by thevaibhav-dixit — APPROVED** [review]

**Pull request merged**